### DISCUSSION TITLE
laravel sail for older laravel versions

### DISCUSSION THREAD

**Discussion by avrahamm** [laravel/framework#37324] [Q&A]
https://laravel.build/my-app
gives bash script to create  sail container for latest laravel 8 version.

I took this script and tried to customize it for 7.* version, all same except this:

docker run --rm \
    -v $(pwd):/opt \
    -w /opt \
    laravelsail/php80-composer:latest \
    bash -c "**composer create-project laravel/laravel my-app \"7.*\"** && cd my-app && php ./artisan sail:install --with=mysql"
   
It completes but then 
/my-app$ ls ./vendor/bin/sail
ls: cannot access './vendor/bin/sail': No such file or directory
sail fails to install.

I tried to use laravelsail/php74-composer image from 
https://hub.docker.com/search?q=laravelsail&type=image
yet it didn't succeed.

Any ideas?

**Comment by avrahamm** [comment]
After trying again error messages point that sail's dependencies are not get satisfied by laravel 7 version. More reading clarifies, that unfortunately current sail is supposed to work with Laravel 8 only.

**Comment by madakaheri** [comment]
omg
